api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/point/hoverStyle/size.md

```diff
@@ -5,7 +5,7 @@ default: undefined
 ---
 ---
 ##### shortDescription
-Specfies the diameter of series points in the hovered state.
+Specfies the diameter of series points in hovered state.
```

Comment:
It should be either 'a hovered state' (if there is more than one hovered state) or 'the hovered state'